code/modules/species/species.dm

```diff
@@ -412,8 +412,22 @@ The slots that you can use are found in items_clothing.dm and are the inventory
 		if(FEMALE)
 			t_him = "her"
 
-	H.visible_message("<span class='notice'>[H] hugs [target] to make [t_him] feel better!</span>", \
-					"<span class='notice'>You hug [target] to make [t_him] feel better!</span>")
+	var/zonefound = FALSE //Used as a hacky default statement, since the cases require extra checks to make sure we aren't interacting with a missing limb
+	if(ishuman(target))
+		var/mob/living/carbon/human/U = target
+		switch(H.zone_sel.selecting)
+			if(BP_R_HAND)
+				if(U.has_organ(BP_R_HAND))
+					H.visible_message(SPAN_NOTICE("\The [H] shakes \the [U]'s hand."), SPAN_NOTICE("You shake \the [U]'s hand."))
+					zonefound = TRUE
+			if(BP_L_HAND)
+				if(U.has_organ(BP_L_HAND))
+					H.visible_message(SPAN_NOTICE("\The [H] shakes \the [U]'s hand."), SPAN_NOTICE("You shake \the [U]'s hand."))
+					zonefound = TRUE
+			//here is where tail entwining will go one day
```

Comment:
Please remove this. comment. It has the subtlety of a brick, and if it gets added later, we can worry about it then. :v

Reply:
nuuuuu

Reply:
Yes, please. I can't tell if this is a meme PR or not as it is, and this comment makes me feel like it probably is.

Reply:
The underlying content is not a joke, I'm just capitalizing on the "hand holding illegal" meme. I'll remove it.